websites/R/Rumble/presence.ts

```diff
@@ -0,0 +1,193 @@
+const presence = new Presence({
+		clientId: "1090817608154108046",
+	}),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	}),
+
+ Assets = {
+	Rumble: "https://i.imgur.com/97X6RHQ.png",
+	Live: "https://i.imgur.com/LOh4SHJ.png",
+};
+presence.on("UpdateData", async () => {
+	// setup presence data
+	const presenceData: PresenceData = {
+		largeImageKey: Assets.Rumble,
+	};
+
+	// if page is a video
+	if (
+		document
+			.querySelectorAll("meta")[5]
+			.content.startsWith("https://rumble.com/v")
```

Comment:
Why not use `pathname`? Also, if using data from specific meta tags, try to use more specific selectors.
```ts
document.querySelector<HTMLMetaElement>("meta[property='og:url']")
```